include/mega/crypto/sodium.h

```diff
@@ -59,18 +57,20 @@ class MEGA_API EdDSA
      * @param msg The message to sign.
      * @param msglen Length of the message.
      * @param sig Buffer to take the signature.
+     * @param siglen Size of the buffer to take the signature.
      * @return Number of bytes for signed message (msg length + signature),
      *     0 on failure.
      */
-    int sign(unsigned char* msg, unsigned long long msglen, char* sig);
+    int sign(const unsigned char* m, const unsigned long long mlen,
```

Comment:
Can we please use more meaningful parameter/variable names? Also, the parameters don't match those in the Doxygen string above.